snuba/datasets/events_processor.py

```diff
@@ -30,6 +30,9 @@ def extract_user(output, user):
     output['user_id'] = _unicodify(user.get('id', None))
     output['username'] = _unicodify(user.get('username', None))
     output['email'] = _unicodify(user.get('email', None))
+    # N.b.: IP address may be a hash of the real IP address (done by PII
+    # stripping). For as long as we can store arbitrary strings in Clickhouse,
+    # let's index those hashes as well.
```

Comment:
what do we gain in preserving those hashes ?
That would reduce our data integrity, so I think we should have a good reason for doing this.

Reply:
If you store a hash of an IP address you might have anonymized the IP sufficiently for compliance while still being able to count distinct users

I am not sure if people do this fwiw

Reply:
I'd prefer this hash to be in another column if we keep storing it. So we do not break the assumption that the ip_address column actually contains an ip_address.